napari/layers/base/base.py

```diff
@@ -76,14 +79,14 @@ class Layer(KeymapProvider, ABC):
     z_index : int
         Depth of the layer visual relative to other visuals in the scenecanvas.
     coordinates : tuple of float
-        Coordinates of the cursor in the data space of each layer. The length
-        of the tuple is equal to the number of dimensions of the layer.
+        Cursor position in data coordinates.
     corner_pixels : array
         Coordinates of the top-left and bottom-right canvas pixels in the data
-        space of each layer. The length of the tuple is equal to the number of
-        dimensions of the layer.
-    position : 2-tuple of int
-        Cursor position in the image space of only the displayed dimensions.
+        coordinates of each layer. For multiscale data the coordinates are in
+        the space of the currently viewed data level, not the highest resolution
+        level.
+    position : tuple
+        Cursor position in world coordinates.
```

Comment:
@sofroniewn can come in a later PR (this is amazing) but what do you think about coordinates -> data_coordinates, and position -> world_position/real_position/similar? Currently "coordinates" and "position" is confusing to me without coming over to read this.

Reply:
yes! I want that, probably in the next PR or so, but before `0.4.0`. That will be a stronger backwards compatibility break, so I want to put it in its own thing. Ultimately I think we want to pull this off the layer entirely!